### PR TITLE
Feature/compression

### PR DESCRIPTION
**Pull request by toughIQ**
Hi,
since I was missing the COMP-LZO compression setting in openvpn.conf I added a config parameter "-z" to enable compression if desired.
If compression is set in server, this parameter is also set in client config files.
Bye, Chris

### CODE REVIEW

**Comment by ypid** [comment]
@kylemanna Could you consider merging #67 instead to solve this once and for all?

File: bin/ovpn_genconfig [hunk 1]
```diff
@@ -74,11 +75,12 @@ OVPN_TLS_CIPHER=''
 OVPN_CIPHER=''
 OVPN_AUTH=''
 
+
 # Import defaults if present
 [ -r "$OVPN_ENV" ] && source "$OVPN_ENV"
 
 # Parse arguments
-while getopts ":a:C:T:r:s:du:cp:n:DNm:t" opt; do
+while getopts ":a:C:T:r:s:du:cp:n:DNm:t:z" opt; do
```
Comment (kylemanna) [inline bin/ovpn_genconfig]: The `:` here is wrong.  Can you replace it and rebase the pull request?

**Comment by toughIQ** [comment]
Hi,
since I am new to working with GIT I hope I did it right. Changed the :z and rebased.
Bye, Chris

**Comment by kylemanna** [comment]
Rebased and merged

**Pull request closed without merging**